python/lsst/analysis/tools/actions/plot/gridPlot.py

```diff
@@ -86,12 +86,25 @@ class GridPlot(PlotAction):
         doc="String arguments passed into fig.suptitle() defining the figure title.",
         optional=True,
     )
+    xlabel = Field[str](
+        doc="String argument passed into fig.supxlabel() defining the figure x label.",
+        optional=True,
+    )
+    ylabel = Field[str](
+        doc="String argument passed into fig.supylabel() defining the figure y label.",
+        optional=True,
+    )
```

Comment:
Can we make these xAxisLabel and yAxisLabel to bring them inline with other plots?